kitchen-calculator/lib/kitchen_calculator.ex

```diff
@@ -1,6 +1,6 @@
 defmodule KitchenCalculator do
-  def get_volume(volume_pair) do
-    elem(volume_pair, 1)
+  def get_volume({volumn, quantity}) do
```

Comment:
The first element of the tuple can be ignored. I'm pretty sure you're getting a compilation warning. Also, you have a typo there.